sklearn/cross_decomposition/tests/test_pls.py

```diff
@@ -83,25 +84,40 @@ def check_ortho(M, err_msg):
         [[-0.61330704,  0.25616119, -0.74715187],
          [-0.74697144,  0.11930791,  0.65406368],
          [-0.25668686, -0.95924297, -0.11817271]])
-    assert_array_almost_equal(pls_ca.x_weights_, x_weights)
+    # x_weights_sign_flip holds columns of 1 or -1, depending on sign flip
+    # between R and python
+    x_weights_sign_flip = pls_ca.x_weights_ / x_weights
 
     x_rotations = np.array(
         [[-0.61330704,  0.41591889, -0.62297525],
          [-0.74697144,  0.31388326,  0.77368233],
          [-0.25668686, -0.89237972, -0.24121788]])
-    assert_array_almost_equal(pls_ca.x_rotations_, x_rotations)
+    x_rotations_sign_flip = pls_ca.x_rotations_ / x_rotations
 
     y_weights = np.array(
         [[+0.58989127,  0.7890047,   0.1717553],
```

Comment:
OK - still weird that + is on some of them but not others. I can deal with it though